src/Translation/English/translation.json

```diff
@@ -351,21 +350,19 @@
   "signIn": {
     "back": "Back",
     "signUp": "Sign up",
-    "existingUserTitle": "Log in to Portato",
-    "existingUserText": "Log in with your email address and password at Portato.",
+    "existingUserTitle": "Log in",
     "nonExistingUserTitle": "Don't have a profile yet?",
     "nonExistingUserText": "Create a profile in just a few steps and become part of the Portato community.",
```

Comment:
"Create a profile in just a few steps and become part of the Portato community.",

-> Maybe this could simply be `Create a profile and become part of the Portato community.`?